stories/plot-object.stories.tsx

```diff
@@ -16,8 +16,13 @@ export function Control() {
       margin: { bottom: 50, left: 50, top: 5, right: 10 },
     },
     axes: {
-      x: { label: 'My X axis', position: 'bottom', min: 0, max: 10 },
-      y: { label: 'My Y axis', position: 'left', min: 0, max: 10 },
+      x: {
+        label: 'My X axis',
+        position: 'horizontal' as const,
```

Comment:
what happens without the `as const` ?

Reply:
something like `string is not assignable to 'horizontal' | 'vertical'`

Reply:
we need to find another way. I don't want to have to do this everytime I use the object API

Reply:
what do you think of using enums?

```ts
const enum Positions {
  horizontal = 'horizontal',
  vertical = 'vertical',
}

{ y: { position: Positions.vertical } }
```

Reply:
strings are fine. I think you just need to specify the type when you create the plot object: `const plot: PlotObjectType = { ... }`